master/internal/core.go

```diff
@@ -910,7 +910,6 @@ func (m *Master) Run(ctx context.Context) error {
 	)
 	tasksGroup := m.echo.Group("/tasks", authFuncs...)
 	tasksGroup.GET("", api.Route(m.getTasks))
-	tasksGroup.GET("/:task_id", api.Route(m.getTask))
```

Comment:
removed this endpoint since as far as I can tell it was broken for ~13 months

Reply:
we should deprecate `/tasks` as well - I believe it's still being used by the CLI though.